src/dataLoaders/components/collectorsWizard/verify/PluginConfigSwitcher.tsx

```diff
@@ -0,0 +1,143 @@
+// Libraries
+import React, {FC, useState} from 'react'
+import Loadable from 'react-loadable'
+import {connect, ConnectedProps} from 'react-redux'
+
+// Components
+import {Overlay} from '@influxdata/clockface'
+import {PluginCreateConfigurationFooter} from 'src/writeData/components/PluginCreateConfigurationFooter'
+
+// Types
+import {AppState} from 'src/types'
+// Actions
+import {notify as notifyAction} from 'src/shared/actions/notifications'
+import {
+  clearSteps,
+  decrementCurrentStepIndex,
+  incrementCurrentStepIndex,
+  setCurrentStepIndex,
+  setSubstepIndex,
+} from 'src/dataLoaders/actions/steps'
+import {clearDataLoaders} from 'src/dataLoaders/actions/dataLoaders'
+
+// Constants
+import {BUCKET_OVERLAY_WIDTH} from 'src/buckets/constants'
+
+const PLUGIN_CREATE_CONFIGURATION_OVERLAY_DEFAULT_WIDTH = 1200
+const PLUGIN_CREATE_CONFIGURATION_OVERLAY_OPTIONS_WIDTH = 480
+const spinner = <div />
```

Comment:
should this be a spinner container?

Reply:
This is incase the next page takes some time to load, its also being used in the PluginCreateConfigurationWizard.tsx that timmy wrote. if we remove it here we should also remove it there and vise versa